src/wrapper/errors.rs

```diff
@@ -66,6 +66,9 @@ pub enum Error {
         #[source]
         source: CharTryFromError,
     },
+
+    #[error("JNI >= 1.2 for ExceptionCheck API")]
```

Comment:
Yeah, I can tweak this error message to improve it as suggested.

For the bytebuffer APIs they now do runtime version checks with:

`self.ensure_version(JNI_VERSION_1_4)?;`

but it might be better to just set a higher bar - requiring JNI 1.4 should be completely reasonable too.

Reply:
For reference here; I bumped the version checked in `JNIEnv::from_raw()` to 1.4 so the runtime checks for 1.4 could be removed for the direct byte buffer APIs.